test/unit/console_log_utils_test.dart

```diff
@@ -130,6 +131,19 @@ void main() {
         expect(printCalls, ['foo', 'bar']);
       });
 
+      test('prints logs that use formatter syntax', () {
+        // This also tests functionally that the print call occurs in the right zone,
+        // which need to happen for them to be forwarded in the terminal in a test environment.
+        // If it wasn't in the right zone, we wouldn't be able to record it, and neither would
+        // the test package.
+        final printCalls = recordPrintCalls(() {
+          printConsoleLogs(() {
+            callMethod(getProperty(window, 'console'), 'log', ['%s World Number %d! %j', 'Hello', 5, jsify({'doWeComeInPeace':false}), 'additional']);
```

Comment:
Can we also add a case without `'additional'` so we have test coverage for that codepath?

Reply:
you mean like when there isn't enough args?

Reply:
That's a good case too, but I was thinking when there are exactly the number of args to match the format placeholders, like this case without `'additional'`

Reply:
wow